examples/bench_cudamat.py

```diff
@@ -1,11 +1,12 @@
+from __future__ import print_function, division
```

Comment:
`print_function` requires Python 2.6+. Was `cudamat` compatible to earlier versions before? (On the other hand, this does not affect the main library, only the example and benchmark scripts.)